Examples/2048/2048TV.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,429 @@
+// !$*UTF8*$!
+{
```

Comment:
They are identical to the corresponding iOS projects except for a few lines... maybe there's a way they could be generated from a single file, or combined.